src/commands/game.ts

```
@@ -1,50 +1,64 @@
+import * as z from 'zod'
```

ESLint

[error] Unable to resolve path to module 'zod'.

(import-x/no-unresolved)